documentation/install.rst

```diff
@@ -138,6 +138,24 @@ For Ubuntu18+ run, an example for LLVM/Clang 10:
     sudo apt-get -y install  clang-10 llvm-10 libclang-10-dev llvm-10-dev
     sudo apt-get -y install  cmake make gcc g++
 
+For MacOSX:
+
+  Make sure the XCode is installed. If needed, install cmake, python and other utilities, e.g. using homebrew:
+  
+  .. code-block:: console
+    
+    brew install wget coreutils xz pybind11 cmake
```

Comment:
Upstream Pybind11 should not be used, instead please point user to use https://github.com/RosettaCommons/pybind11 . It have master that is in lock-step with Binder sources so it should always work correctly.